examples/v1beta1/pipeline/realtimeimageclassfication/steps.md

```diff
@@ -0,0 +1,72 @@
+### Real time Image Classification Use-Case : End to End Pipeline
```

Comment:
Can we have a link from README to this doc ? 
Or, as I think, it's better to add these steps directly to [README](https://github.com/kubeflow/katib/blob/fa344717d717e5bbafbc5e40b0d1c6aa0ea2366c/examples/v1beta1/pipeline/realtimeimageclassfication/README.md)

Reply:
Pushed all the required changes please review that @andreyvelich